stable/anchore-engine/README.md

```diff
@@ -244,6 +244,32 @@ See the anchore-engine [CHANGELOG](https://github.com/anchore/anchore-engine/blo
 A Helm post-upgrade hook job will shut down all previously running Anchore services and perform the Anchore database upgrade process using a Kubernetes job. 
 The upgrade will only be considered successful when this job completes successfully. Performing an upgrade will cause the Helm client to block until the upgrade job completes and the new Anchore service pods are started. To view progress of the upgrade process, tail the logs of the upgrade jobs `anchore-engine-upgrade` and `anchore-enterprise-upgrade`. These job resources will be removed upon a successful Helm upgrade.
 
+## Chart version 1.17.0
+
+Chart version 1.17.0 is a tightly scoped release, specifically aimed at enabling the Grype DB Builder within the Anchore Enterprise Feeds service. This upgrade will allow users to easily switch from the legacy vulnerability provider to the next-gen Grype vulnerability provider, without any vulnerability feed downtime. The impacts of this upgrade are as follows:
```

Comment:
```suggestion
Chart version 1.17.0 is a purely Enterprise focused release. Anchore Engine users will see no change in behavior from this release.

For Enterprise users, this release specifically helps reduce downtime needed during the transition from the v1 scanner to the v2 scanner. This version sets the GrypeDB driver to run in the feed service v1-scanner deployments so that the GrypeDB is ready when the update to the v2 scanner is made and thus reduces effective downtime during the maintenance window needed for that configuration change.

It is recommended that users upgrade to this chart version prior to changing the scanner configuration to move from the V1 scanner to the V2 scanner.
```
Then insert the resource warning here before going into the bullet points about config options.